src/cntk-model/src/main/python/CNTKModel.py

```diff
@@ -12,8 +12,10 @@
 @inherit_doc
 class CNTKModel(_CNTKModel):
     """
-    :param SparkSession SparkSession: The SparkSession that will be used to find the model
-    :param str location: The location of the model, either on local or HDFS
+
+    Args:
```

Comment:
Yes. I found a sphinx extension that will take "Google Style" (also, "numpy style") and render them in the same format as :param: I opted to change the documents to this style because it makes them more readable as "classic python" for commandline help without losing the nicer API documentation style